planetiler-basemap/src/main/java/com/onthegomap/planetiler/basemap/util/LanguageUtils.java

```diff
@@ -53,10 +54,22 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * <a href="https://github.com/openmaptiles/openmaptiles-tools/blob/master/sql/zzz_language.sql">openmaptiles-tools</a>.
  */
 public class LanguageUtils {
+  // See https://github.com/onthegomap/planetiler/issues/86
+
+  // Name tags that should be eligible for finding a latin name.
+  // See https://wiki.openstreetmap.org/wiki/Multilingual_names
+  private static final Predicate<String> VALID_NAME_TAGS =
+    Pattern.compile("^name:[a-z]{2,3}(-[a-z]{4})?([-_][a-z]{2,})?(-[a-z]{2})?$", Pattern.CASE_INSENSITIVE)
```

Comment:
Technically, BCP&nbsp;47 allows a UN&nbsp;M49 region code in place of an ISO&nbsp;3166-1 country code, but [the only extant occurrences](https://overpass-turbo.eu/s/1h7S) are mere coincidences, all of them instances of mistagging.

The `abc-x-extension` syntax is also permitted for unregistered extensions, but [`name:fr-x-gallo`](https://taginfo.openstreetmap.org/keys/name%3Afr-x-gallo) is the only key with non-negligible use, for [Gallo](https://en.wikipedia.org/wiki/Gallo_language).

Reply:
Thanks! Added those cases and they pass now.  The transliteration improvements will be a larger project, will need to tackle that in a separate change later.